README.md

```diff
@@ -22,12 +22,12 @@ Read more about contributing to Dark Reader in [CONTRIBUTING.md](https://github.
 
 You can install the extension from a file.  
 Install [Node.js LTS](https://nodejs.org/en/). Download the source code (or check out from git).  
-Open terminal in the root folder and run:  
+Open the terminal in the root folder and run:
 
-- `npm install`  
-- `npm run build`  
+- `npm install`
+- `npm run build`
 
-This will generate a `build/release/darkreader-chrome.zip` file that is useable in a Chromium-based browser and also a `build/release/darkreader-firefox.xpi` file that is useable in Firefox.
+A `build/release/darkreader-chrome.zip` file will generate for use in a Chromium-based browser, and also a `build/release/darkreader-firefox.xpi` file will generate for use in Firefox.
```

Comment:
```suggestion
A `build/release/darkreader-chrome.zip` file will generated for use in a Chromium-based browser, and also a `build/release/darkreader-firefox.xpi` file will generated for use in Firefox.
```

Reply:
The word "generate" is the correct tense form of the word to use here.

The word "generated" is the past tense form of the word "generate".

If we were to use "generated" here, then the correct wording would be "be generated". However, doing that invokes passive voice construction. It is always best to use active voice construction to provide a clear, concise, and straight to the point description of what is happening.

Reply:
That was my explanation and thoughts on the wording. What are your thoughts on the wording after my explanation?

How do you wish to proceed? I can change it if needed.

Reply:
I think we should change the whole sentence structure, because the current form be generated isn't that good and just `generate` feels odd to me(it seems to be technically correct, but it doesn't feel like it).

Reply:
How about now? I took the original sentence and reworded it/reduced the wordiness. I believe this is the best of both worlds and perhaps something we can agree on? 🤣